implementation/UserValidation.py

```
@@ -1,13 +1,14 @@
+import requests
 class UserValidation:
-        def __init__(self,cursor):
-                self.cursor=cursor
+        def __init__(self):
+               
                 self.currentUser=None 
                 self.doc=None 
-        def userValidate(self,mail:dict): 
-               return self.cursor.collection('user_model_table').document(mail)
+        def userValidate(self,mail:str): 
+               result=requests.get(f"http://localhost:7006/fetch/{mail}").json()        
```

Flake8 (7.2.0)

[error] indentation is not a multiple of 4

(E111)